package/DESCRIPTION

```diff
@@ -0,0 +1,32 @@
+Package: shinyMlr
+Title:  A graphical user interface for machine learning in R
+Description:
+  shinyMlr wraps the functionalities of the R-package mlr
+  into a graphical user-interface built with shiny.
+  This enables the user to conduct all steps of the machine learning workflow from his browser.
+Authors@R: c(
+  person("Florian", "Fendt", email = "flo_fendt@gmx.de",
+    role = c("aut", "cre")),
+  person("Stefan", "Coors", email = "stefan.coors@gmx.net",
+    role = "aut"))
+URL: https://github.com/mlr-org/shinyMlr
+BugReports: https://github.com/mlr-org/shinyMlrissues
+License: BSD_2_clause + file LICENSE
+Encoding: UTF-8
+Roxygen: list(wrap = FALSE)
+Depends:
+  R (>= 3.3.2)
+Imports:
+  shiny,
+  shinythemes
+Suggests:
+  mlr,
+  testthat,
+  RSelenium,
+  roxygen2,
+  knitr,
```

Comment:
Now the whole app is in the package we should also add mlr-imports as suggests here, right?